.devcontainer/postCreateCommand.py

```
@@ -0,0 +1,199 @@
+    
+    files = list(base_dir.glob(pattern))
+    for file in files:
+        if file.is_file():
+            file.chmod(0o755)
+            print_success(f"Made {file.relative_to(Path.cwd())} executable")
+
+
+def command_exists(cmd: str) -> bool:
+    """Check if a command exists in PATH"""
+    try:
+        subprocess.run(
```

Ruff (0.14.4)

`subprocess` call with `shell=True` identified, security issue

(S602)